extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/PersistentLoginManager.java

```diff
@@ -135,7 +138,8 @@ public void save(String value, RoutingContext context, String cookieName, Restor
             message.put(encrypted);
             String cookieValue = Base64.getEncoder().encodeToString(message.array());
             context.addCookie(
-                    Cookie.cookie(cookieName, cookieValue).setPath("/").setSecure(secureCookie).setHttpOnly(httpOnlyCookie));
+                    Cookie.cookie(cookieName, cookieValue).setPath("/").setSameSite(cookieSameSite).setSecure(secureCookie)
```

Comment:
Careful with the path, when applications define a "public" facing zone and a "secure" zone, using a path, say for example, `/public/static-html` and `/secure/api` setting the path to `/` means cookies can be exfiltrated. It would be better  to make this configurable as the framework cannot predict the developer intentions.

Reply:
@pmlopes can you confirm your concern is addressed by https://github.com/quarkusio/quarkus/pull/30171 ?